doc/api/path.markdown

```diff
@@ -117,6 +119,20 @@ path.format({
 '/file.txt'
 ```
 
+An example on Windows:
+
+```js
+path.format({
+    root : "C:\\",
+    dir : "C:\\path\\dir",
+    base : "file.txt",
+    ext : ".txt",
+    name : "file"
+})
+// returns
+'C:\\path\\dir\\file.txt'
```

Comment:
@mscdex How it is done here is how it is for the other examples in the doc. This is unfortunate in my opinion and I opened a pull request to change it, but the reviews have been mixed. I invite you to comment at https://github.com/nodejs/node/pull/5670